dali_mqtt_daemon.py

```diff
@@ -202,11 +216,12 @@ def on_connect(
         MQTT_DALI2MQTT_STATUS.format(mqqt_base_topic), MQTT_AVAILABLE, retain=True
     )
     lamps = dali_scan(driver_object, max_lamps)
+    lamps_summary_list = []
     for lamp in lamps:
         try:
-            actual_level = driver_object.send(
-                gear.QueryActualLevel(address.Short(lamp))
-            )
+            short_address = address.Short(lamp)
+            actual_level = driver_object.send(gear.QueryActualLevel(short_address))
```

Comment:
Here should be comment with information that "dali actual_level" is brightness.

Reply:
Probably is better change variable name `actual_level` to `brightness_level` than creating comment